### PR TITLE
Added dummy topics and topic thread pages

### LINKED ISSUE #2: Add something for the front-end
We need to have something for the front-end, at least as a placeholder

<details>
<summary>Don't open this</summary>
nooooooooooooooooooooooooooooooooo
</details>

### PR DESCRIPTION
**Pull request by David-Render**
Fixes #2

* Added box styles for Op post
* Added posts with dummy text and images

### CODE REVIEW

**Comment by David-Render** [comment]
cc @RenderMichael

File: src/FrontEnd/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TopForum</title>
+</head>
+<body>
+    <h1>Hello!</h1>
```
Comment (RenderMichael) [inline src/FrontEnd/index.html]: can you change this from `Hello!` to `Welcome to Top Forum!`?

```suggestion
    <h1>Welcome to Top Forum!</h1>
```

**Review by RenderMichael — CHANGES_REQUESTED** [review]
Looks nice, small suggestion.

**Comment by David-Render** [comment]
@RenderMichael PTAL

**Review by RenderMichael — APPROVED** [review]
thanks! LGTM!

**Pull request merged**